### PR TITLE
Feature/cart

### PR DESCRIPTION
**Pull request by Nikhil-Nirmal**
Added Required changes & Forgot pass page.

### CODE REVIEW

**Review by samflab — APPROVED** [review]

File: Css/login.css [hunk 1]
```diff
@@ -146,17 +146,18 @@ a {
 }
 
 .login input {
-    margin: 1.5rem 0px;
-    padding: 0.5rem 1rem;
+    margin: 0rem 0px;
```
Comment (shraddha-amber) [inline Css/login.css:149]: suggestion: no need to write unit for 0, also if you want to use consistent units

**Review by SubhamChoudhury — APPROVED** [review]

File: Css/login.css [hunk 2]
```diff
@@ -146,17 +146,18 @@ a {
...
+    padding: 1rem 1rem;
```
Comment (shraddha-amber) [inline Css/login.css:150]: could've just given 1rem and it would've applied to all four sides

File: cart.css [hunk 3]
```diff
@@ -153,22 +153,27 @@ a {
...
     display: block;
+    color: #ffffff;
 }
 
 .btn-success {
-    color: #f1f1f1;
+    color: #ffffff;
     padding: 0.4rem;
-    margin: 1.5rem 15px 0px 0px;
+    margin: 1.5rem 15px 0px 2rem;
```
Comment (shraddha-amber) [inline cart.css:176]: its recommended to use consistent units

File: Css/login.css [hunk 4]
```diff
@@ -172,6 +173,15 @@ label {
...
+    color: #ff2d2d;
+}
+
+.forgot-pass-link:hover {
+    color: #3B82F6;
+    text-decoration: underline;
+}
+
 .signup{
     font-family: 'Lato', sans-serif;
```
Comment (shraddha-amber) [inline Css/login.css:186]: If you are not using seperate fonts for headings then try to add this in global css file

**Review by shraddha-amber — APPROVED** [review]

**Pull request merged**